### DISCUSSION TITLE
How to use #[cfg] macro ot specify the minimum qt version for a function

### DISCUSSION THREAD

**Discussion by Ayush1325** [woboq/qmetaobject-rs#241] [Q&A] [answered]
I am trying to wrap a function that was added in Qt 5.12. #239 

I added the `#[cfg(qt_5_12)]` but that doesn't seem to be the correct way since the test fails for QT 5.9.9. So what is the correct way to include a function when the Qt version is higher than 5.12?

**Comment by ogoffart** [comment, marked as answer]
you need to add a `#ifdef`  in the C++ code
